registry/handlers/app.go

```diff
@@ -244,6 +244,8 @@ func NewApp(ctx context.Context, config *configuration.Configuration) *App {
 				options = append(options, storage.ManifestURLsDenyRegexp(re))
 			}
 		}
+	} else {
+		options = append(options, storage.ManifestURLsAllowRegexp(regexp.MustCompile("^$")))
```

Comment:
Can we refactor this later but at least agree on whether we should make this change or now